### ISSUE TITLE
[Bug]: FocusVisible polyfill is not added for sp-toast

### ISSUE THREAD

**Issue by deepthi-majji** [adobe/spectrum-web-components#3480]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Sp-Toast

### Expected behavior

The focus ring should be visible around the clear button inside the toast when the user is navigating with the keyboard in browsers that don't support :focus-visible. 

### Actual behavior

The focus ring is not shown around the clear button inside the toast when the user is navigating with the keyboard in browsers that don't support :focus-visible. 

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
